internal/gitstrap/utils.go

```diff
@@ -20,3 +24,10 @@ func (g *Gitstrap) resolveOrg(m *spec.Model) string {
 	}
 	return m.Metadata.Owner
 }
+
+func (g *Gitstrap) getTeamOwner(m *spec.Model) (string, error) {
+	if m.Metadata.Owner == "" {
+		return "", fmt.Errorf("Team owner is not specified")
```

Comment:
@OrlovM it's recommended to have an `error` variable for such cases: client code will be able to check error with `errors.Is()` or `errors.As()`. See: https://blog.golang.org/go1.13-errors